azurecontainerapps/azurecontainerapps.ts

```diff
@@ -1,18 +1,31 @@
+import * as fs from 'fs';
 import * as path from 'path';
 import * as tl from 'azure-pipelines-task-lib/task';
-import * as fs from 'fs';
-import { Utility } from './src/Utility';
-import { ContainerAppHelper } from './src/ContainerAppHelper';
 import { AzureAuthenticationHelper } from './src/AzureAuthenticationHelper';
+import { ContainerAppHelper } from './src/ContainerAppHelper';
 import { ContainerRegistryHelper } from './src/ContainerRegistryHelper';
+import { TelemetryHelper } from './src/TelemetryHelper';
+import { Utility } from './src/Utility';
 
 const util = new Utility();
 
 export class azurecontainerapps {
 
     public static async runMain(): Promise<void> {
+        let disableTelemetry = false;
+        try {
+            disableTelemetry = tl.getBoolInput('disableTelemetry', false);
```

Comment:
I'm just curious: why do we need to wrap this one in a `try {}` block? I don't think we're doing that for the other tl.getXYZ calls. Is there something specific that makes it more likely to throw?

Reply:
I'm doing this since the argument is a `boolean` rather than a `string`, which uses the `tl.getInput` call vs. the `tl.getBoolInput` call here -- this is just a failsafe to ensure that if a non-`boolean` value is provided, we default to setting `disableTelemetry` to `false` -- what do you think?

Reply:
Yes OK, sounds good. Did you have a chance to test that it would indeed throw if a non bool is passed? The doc isn't very clear about that as far as I can tell, even though I assume it would be the case: https://github.com/microsoft/azure-pipelines-task-lib/blob/master/node/docs/azure-pipelines-task-lib.md#taskgetBoolInput

Reply:
Actually, looking at the source for [`getBoolInput`](https://github.com/microsoft/azure-pipelines-task-lib/blob/0d2f17bf6a299c0d784a85dbf273974052a70b7b/node/task.ts#L273), it's just doing a simple check to see if the provided input is a case-insensitive version of `true`, otherwise it just returns `false` (since it calls `getInput` under-the-hood, and that method doesn't throw unless the `required` argument is set to `true`, which it's not in our case). We should be OK to remove this `try` block, but I'll double-check with a random value when testing.

Reply:
OK sounds good, thanks